jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1401,7 +1401,7 @@ Function Start-Migration {
     Begin {
         Write-ToLog -Message:('####################################' + (get-date -format "dd-MMM-yyyy HH:mm") + '####################################')
         # Start script
-        $admuVersion = '2.4.2'
+        $admuVersion = '2.4.4'
```

Comment:
Should our version for this module should be 3.5.0? I'd argue that this is new functionality, not solving a bug or addressing a previous issue.

Reply:
I agree with 2.5, it's new functionality